app/main/posts/posts-routes.js

```diff
@@ -115,30 +107,36 @@ function (
     .state(
         {
             name: 'posts.map',
-            url: '/views/map',
-            views: {
-                listView: {
-                    template: function ($state, $transition$) {
-                        return '<post-view-map></post-view-map>';
-                    }
-                }
-            },
+            abstract: true,
+            component: 'postViewMap',
             params: {
                 view: {value: 'map', squash: true},
                 filterState: {value: null, squash: true}
+            }
+        }
+    )
+    .state(
+        {
+            url: '/views/map',
+            name: 'posts.map.all',
+            views: {
+                'mode-context': 'modeContext'
             },
             onEnter: function ($state, $transition$, PostFilters, savedSearch) {
-                if (!savedSearch) {
-                    PostFilters.resetDefaults();
-                }
+                PostFilters.setMode('all');
```

Comment:
we cannot do resetDefaults and setMode here because the user needs to be able to move around and keep their filters.

Reply:
https://github.com/ushahidi/platform-client/pull/931